Packages/vcs/vcs/vcs2vtk.py

```diff
@@ -251,11 +251,18 @@ def genGridOnPoints(data1, gm, deep=True, grid=None, geo=None,
     if geo is None:
         bounds = pts.GetBounds()
         xm, xM, ym, yM = [bounds[0], bounds[1], bounds[2], bounds[3]]
-        # We don't use the zooming feature (gm.datawc) for geographic
-        # projections. We use wrapped coordinates for doing the projection
+        # We use zooming feature (gm.datawc) for linear and ploar projections.
+        # We use wrapped coordinates for doing the projection
```

Comment:
Sure, I'll cleanup the comment. The command for  test_vcs_basic_isofill_-3_proj_gmflip_NH_via_gm.png is

"/home/danlipsa/src/uvcdat/testing/vcs/test_vcs_basic_gms.py" "--gm_type=isofill" "--projection=-3" "--lat1=90" "--lat2=0" "--range_via_gm" "--gm_flips_lat_range" "--source=/home/danlipsa/build/uvcdat/uvcdat-testdata/baselines/vcs/test_vcs_basic_isofill_-3_proj_gmflip_NH_via_gm.png"

so
(90, 0) flipped -> (0, 90).
which means view from the South.
The hole is (-90, 0). Note the NH which is appended to the file is wrong: https://github.com/UV-CDAT/uvcdat/issues/1903